doc/source/tech_note/Methane/CLM50_Tech_Note_Methane.rst

```diff
@@ -286,14 +286,14 @@ The diffusive transport through aerenchyma (*A*, mol m\ :sup:`-2` s\ :sup:`-1`)
 
    A=\frac{C\left(z\right)-C_{a} }{{\raise0.7ex\hbox{$ r_{L} z $}\!\mathord{\left/ {\vphantom {r_{L} z D}} \right. \kern-\nulldelimiterspace}\!\lower0.7ex\hbox{$ D $}} +r_{a} } pT\rho _{r} ,
  
-where *D* is the free-air gas diffusion coefficient (m:sup:`2` s\ :sup:`-1`); *C(z)* (mol m\ :sup:`-3`) is the gaseous
+where *D* is the free-air gas diffusion coefficient (m\ :sup:`2` s\ :sup:`-1`); *C(z)* (mol m\ :sup:`-3`) is the gaseous
 concentration at depth *z* (m); :math:`r_{L}`  is the ratio of root
 length to depth; *p* is the porosity (-); *T* is specific aerenchyma
-area (m:sup:`2` m\ :sup:`-2`); :math:`{r}_{a}` is the
+area (m\ :sup:`2` m\ :sup:`-2`); :math:`{r}_{a}` is the
 aerodynamic resistance between the surface and the atmospheric reference
-height (s m:sup:`-1`); and :math:`\rho _{r}`  is the rooting
+height (s m\ :sup:`-1`); and :math:`\rho _{r}`  is the rooting
 density as a function of depth (-). The gaseous concentration is
-calculated with Henry’s law as described in equation .
+calculated with Henry’s law as described in equation :eq:`24.7`.
```

Comment:
I wonder why all of these were missing in the first place?  Thanks for adding these references.